go-controller/pkg/libovsdbops/meter.go

```diff
@@ -51,9 +51,10 @@ func CreateOrUpdateMeterOps(nbClient libovsdbclient.Client, ops []ovsdb.Operatio
 	for _, band := range meterBands {
 		meter.Bands = append(meter.Bands, band.UUID)
 	}
+	// TODO(trozet): evalute if safe to update all fields of the meter
 	opModel := operationModel{
 		Model:          meter,
-		OnModelUpdates: onModelUpdatesAll(),
+		OnModelUpdates: []interface{}{&meter.Fair, &meter.Unit},
```

Comment:
I think we need the Bands here as well.

With that, it is equivalent to what we had before, as it wouldn't make sense for a meter to exist without bands, and we always provide a meter fairness value.